Scripts/build/Jenkins/Jenkinsfile

```diff
@@ -344,65 +340,85 @@ def PreBuildCommonSteps(Map pipelineConfig, String repositoryName, String projec
                 script: 'python/get_python.bat'
         }
 
-        if(env.CLEAN_OUTPUT_DIRECTORY?.toBoolean() || env.CLEAN_ASSETS?.toBoolean()) {
-            def command = "${pipelineConfig.BUILD_ENTRY_POINT} --platform ${platform} --type clean"
-            if (env.IS_UNIX) {
-                sh label: "Running ${platform} clean",
-                   script: "${pipelineConfig.PYTHON_DIR}/python.sh -u ${command}"
-            } else {
-                bat label: "Running ${platform} clean",
-                    script: "${pipelineConfig.PYTHON_DIR}/python.cmd -u ${command}".replace('/','\\')
-            }
-        }
+        // Always run the clean step, the scripts detect what variables were set, but it also cleans if
+        // the NODE_LABEL has changed
+        def command = "${pipelineConfig.PYTHON_DIR}/python"
+        if(env.IS_UNIX) command += '.sh'
```

Comment:
I am assuming env.IS_UNIX is true for Mac as well?

Reply:
Yup, it just returns true for any unix based OS, which includes Mac